views/templates/front/payment-method.tpl

```diff
@@ -30,7 +30,7 @@
          data-local-payment-method="{$paymentMethodType|escape:'html':'UTF-8'}">
         {if $isPrestaShop16}
         <div class="col-xs-12 col-md-12">
-            <div class="payment_module" style="background-image: url(https://checkoutshopper-live.adyen.com/checkoutshopper/images/logos/medium/{$paymentMethodBrand|escape:'html':'UTF-8'}.png)">
+            <div class="payment_module adyen-collapse" style="background-image: url(https://checkoutshopper-live.adyen.com/checkoutshopper/images/logos/medium/{$paymentMethodBrand|escape:'html':'UTF-8'}.png)">
```

Comment:
isn't it only added when `$collapsePayments`?

Reply:
The new class is just used to apply 1.6 styling to it, not to denote if the actual div is collapsible or not. Maybe renaming it to `adyen-collapse-styling` would make it more clear?

Reply:
@Morerice yes I think it would be a bit better indeed